Description: Patch fdisk and lvm module.info dependencies
 When translated to the control file, the default module dependencies have
 circular dependencies. The dependencies are fixed in the control file itself
 when running the 'buildsrc' script, this patch makes the module.info consistent
 with the control file.
Author: Jeremy Davis <jeremy@turnkeylinux.org>
Forwarded: no
Last-Update: 2025-09-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/modules/fdisk/module.info
+++ b/modules/fdisk/module.info
@@ -2,7 +2,7 @@ name=Partition Manager
 category=hardware
 os_support=*-linux
 desc=Partitions on Local Disks
-depends=mount proc raid 2.402
+depends=mount proc 2.402
 longdesc=Create and edit partitions on local SCSI, IDE and SATA disks on Linux.
 novserver=1
 noopenvz=1
--- a/modules/lvm/module.info
+++ b/modules/lvm/module.info
@@ -2,6 +2,6 @@ name=LVM
 desc=Logical Volume Management
 category=hardware
 os_support=*-linux
-depends=mount fdisk 2.402
+depends=mount fdisk raid 2.402
 longdesc=Configure volume groups, physical volumes and logical volumes for Linux LVM.
 version=2.402
